source/features/vertical-front-matter.tsx

```diff
@@ -4,9 +4,7 @@ import select from 'select-dom';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
-
-// https://github.com/github/markup/blob/cd01f9ec87c86ce5a7c70188a74ef40fc4669c5b/lib/github/markup/markdown.rb#L34
-const hasFrontMatter = (): boolean => pageDetect.isSingleFile() && /\.(mdx?|mkdn?|mdwn|mdown|markdown|litcoffee)$/.test(location.pathname);
+import {isSingleMarkdownFile} from '../github-helpers';
```

Comment:
This isn’t equivalent. Litcoffee isn’t markdown and we need to preserve “has front matter” as its own test

Reply:
Given how it’s rendered, I wonder if the test for all of these features should be called “is markdown file” at all: https://github.com/rmetzler/litcoffee-examples/blob/master/fib.litcoffee

There are more file types that would benefit from `unlink-images-auto-link`, like: https://stackoverflow.com/questions/41112069/what-file-types-does-github-support-for-readmes